x/slashing/client/rest/query.go

```diff
@@ -29,7 +29,23 @@ func registerQueryRoutes(cliCtx context.CLIContext, r *mux.Router) {
 	).Methods("GET")
 }
 
```

Comment:
This should not have been merged @fedekunze. There are typos and the annotations are incorrect.